src/components/Input/components/Select/index.tsx

```diff
@@ -1,20 +1,101 @@
-import React, { ReactElement } from 'react';
+/* eslint-disable max-statements */
+
+import React, { ReactElement, ReactNode, ReactChild } from 'react';
+import clsx from 'clsx';
 import InputBase from '../InputBase';
 import { Props } from '../InputBase/types';
+import DefaultCustomControl from './components/DefaultCustomControl';
+
+
+/**
+ * The select component is the brewkit implementation of the HTML select, with a few
+ * extra features.
+ */
+const Select = (props: Props): ReactElement => {
+
+
+    const {
+        className,
+        children,
+        onChange: onChangeProp,
+        value: valueProp,
+        defaultValue: defaultValueProp,
+        ...otherProps
+    } = props;
```

Comment:
I know @lawren wants to destructure the props in the function declaration instead of like you're doing here. This is correct in the `DefaultCustomControl/index.tsx` component but seemed to just miss it here.

e.g.
```
const Select = ({
    className,
    children,
    onChange: onChangeProp,
    value: valueProp,
    defaultValue: defaultValueProp,
    ...otherProps
}: Props): ReactElement => {